spring-cloud-starter-stream-processor-tcp-client/src/main/resources/META-INF/spring.factories

```diff
@@ -0,0 +1,2 @@
+org.springframework.boot.env.EnvironmentPostProcessor=\
+  org.springframework.cloud.stream.app.tcp.ContentTypeEnvironmentPostProcessor
```

Comment:
New line in the end of each file.
You can configure IDEA to do that for you: https://stackoverflow.com/questions/16761227/how-to-make-intellij-idea-insert-a-new-line-at-every-end-of-file

Reply:
+1